### DISCUSSION TITLE
How to move search result notes at once?

### DISCUSSION THREAD

**Discussion by hunorbatori** [TriliumNext/Trilium#1934] [Q&A] [answered]
Hi!

I'm migrating from Google Keep to Trilium and I successfully imported all my notes but I struggle with organizing them.
So I have many labels in Keep to differentiate between topics and as these labels appear as text in trilium I can easily search for them one by one but I don't know how to move all the search results under a folder at the same time.

I'm aware that there are actions on the search page and I can add labels for example but I would like to use folders to organize my topics instead of labels. Can I do that somehow? The best solution I found is to move the notes one by one into the desired folder but I have several hundreds of notes imported so I would shortcut it if possible.

Thanks!

**Comment by zadam** [comment, marked as answer]
Hi, for that I would use the tree panel:

![image](https://user-images.githubusercontent.com/617641/117869660-59ba7900-b29b-11eb-809f-7d6e314192c5.png)

Expand the search note which contains the list of search results, you can select all the notes in the search note by `CTRL-A` or select individually by holding `shift` and either clicking or moving with cursor keys. Then copy into clipboard with `CTRL-C` and paste into target parent note with `CTRL-V`.

Note however that this does not *move* the notes, it [clones](https://github.com/zadam/trilium/wiki/Cloning-notes) them. You can delete the original location/clone later ...
